boards/arm/nrf5340dk_nrf5340/board.cmake

```diff
@@ -8,6 +8,10 @@ if(CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340DK_NRF5340_CPUAP
 board_runner_args(jlink "--device=nrf5340_xxaa_app" "--speed=4000")
 endif()
 
+if(CONFIG_TFM_FLASH_MERGED_BINARY)
+set_property(TARGET runners_yaml_props_target PROPERTY hex_file "../tfm_merged.hex")
```

Comment:
Could we use a variable with the full path (similar to `${ZEPHYR_BINARY_DIR}`) here? To avoid a relative path that might change in the future.

Reply:
Agreed, will address this, too.